content/id/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Lifecyle Hook pada Kontainer
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Laman ini menjelaskan bagaimana semua Kontainer yang diatur kubelet menggunakan *framework lifecycle hook*
```

Comment:
what about `kubelet` since that's a Kubernetes related terminology?

Reply:
we'll standardize this later, I followed the original english formatting of this page (kubelet without anything)